rest-twitter/TweetRetriever.py

```diff
@@ -0,0 +1,62 @@
+import os
+from birdy.twitter import AppClient, TwitterClientError
```

Comment:
This API requirement should be put in requirement.txt so that at the time of deployment we can run install all the requirements using pip.

Please pull the latest codes and add the birdy to requirements.txt